scvi/dataset/cbmc.py

```diff
@@ -1,32 +1,12 @@
-import pandas as pd
-import numpy as np
+from .csv import CsvDataset
 
-from .dataset import GeneExpressionDataset
 
-
-class CbmcDataset(GeneExpressionDataset):
+class CbmcDataset(CsvDataset):
     url = "https://www.ncbi.nlm.nih.gov/geo/download/" + \
           "?acc=GSE100866&format=file&file=GSE100866%5FCBMC%5F8K%5F13AB%5F10X%2DRNA%5Fumi%2Ecsv%2Egz"
 
     def __init__(self, save_path='data/'):
-        self.save_path = save_path
-
-        # originally: GSE100866_CBMC_8K_13AB_10X-RNA_umi.csv.gz
-
-        self.download_name = 'cbmc.csv.gz'
-
-        expression_data, gene_names = self.download_and_preprocess()
-
-        super(CbmcDataset, self).__init__(
-            *GeneExpressionDataset.get_attributes_from_matrix(
-                expression_data), gene_names=gene_names)
-
-    def preprocess(self):
-        print("Preprocessing cbmc data")
-        expression = pd.read_csv(self.save_path + self.download_name, index_col=0, compression='gzip').T
-        gene_names = np.array(expression.columns, dtype=str)
-
-        selected = np.std(expression.values, axis=0).argsort()[-600:][::-1]
-        expression_data = expression.values[:, selected]
-
-        return expression_data, gene_names
+        super(CbmcDataset, self).__init__(filename='GSE100866_CBMC_8K_13AB_10X-RNA_umi.csv.gz',
+                                          save_path=save_path,
+                                          p_genes=600,
```

Comment:
Why 600? Could we make 600 the default value for a keyword argument to the constructor?

Reply:
Romain chose the 600 genes with highest variance in the paper. Do we want to make 600 the default value? Or will users want to include all genes for most of the cases? 

`brain_small` is similar - I passed in `p_genes=3000` in the constructor 😊

Reply:
Yes, please make 600 the default value to a keyword argument taken by the constructor.

Reply:
Do we also make 3000 the default value for `loom.py` (the same issue for `brain_small`)?